Sources/Plasma/CoreLib/hsSystemInfo.cpp

```diff
@@ -114,6 +115,39 @@ static inline void ICPUID(cpuid_t* info, int function_id)
 #endif
 }
 
+#ifdef HS_BUILD_FOR_APPLE
+static inline ST::string IGetAppleCPUVendor()
+{
+    ST::string result;
+
+#ifdef HAVE_SYSCTL
+    size_t bufsize = 100;
+    char buffer[bufsize];
+
+    if (sysctlbyname("machdep.cpu.brand_string", &buffer, &bufsize, nullptr, 0) == 0) {
```

Comment:
Do we know that `sysctlbyname` is always available when `sysctl` is? `HAVE_SYSCTL` is only testing the latter and not the former, IIRC.

Reply:
`sysctlbyname` exists in macOS 10.0, so I think that's probably safe to depend on